e2e/setup/auth.setup.ts

```diff
@@ -5,13 +5,13 @@ setup("authenticate as admin", async ({ page }) => {
 
   await page.getByRole("link", { name: "Probeer gratis" }).click();
 
-  await page.waitForURL(/account-acc.uitid.be\/*/);
+  await page.waitForURL(/account-keycloak-acc.uitid.be\/*/);
```

Comment:
Shouldn't this be configurable auth0 or keycloak?

Reply:
The acc environment has already been integrated with Keycloak. Is it necessary?